featurebyte/sql/tile_monitor.py

```diff
@@ -28,6 +30,8 @@ async def execute(self) -> None:
         """
         Execute tile monitor operation
         """
+        if not int(os.environ.get("FEATUREBYTE_TILE_MONITORING_ENABLED", "0")):
```

Comment:
could we add a comment as to why we're choosing to do this?

Reply:
Sure, will add a comment. The reason is that one of the bottlenecks we're seeing is caused by the use of merge into commands in the tile monitoring procedures. Since that functionality is not user facing yet, @kchua78 suggested to temporarily disable it for now and only enable it after we resolved the issues.